src/main/scala/org/kframework/kore/Default.scala

```diff
@@ -0,0 +1,122 @@
+package org.kframework.kore
+
+object default {
```

Comment:
Let's not expose these case classes at all. We can make then all private so that users will be required to use the `default.DefaultBuilders` (which we could rename to `default.Builders`) and kore interfaces. This way we don't end up with client code depending directly on the default implementations.